src/WebJobs.Script.WebHost/Models/HostStatus.cs

```diff
@@ -33,6 +33,18 @@ public class HostStatus
         [JsonProperty(PropertyName = "versionDetails", DefaultValueHandling = DefaultValueHandling.Ignore)]
         public string VersionDetails { get; set; }
 
+        /// <summary>
+        /// Gets or sets the Antares version. This corresponds to platform_version on linux and website_platform_version on windows.
+        /// </summary>
+        [JsonProperty(PropertyName = "platformVersion", DefaultValueHandling = DefaultValueHandling.Ignore)]
```

Comment:
Not sure which of these new settings (if any) will actually return anything on Linux Consumption. @balag0 would know

Reply:
PlatformVersion can be made available for Consumption ( available on Appservice plan already). 
InstanceId will remain empty for consumption and we should use CONTAINER_NAME instead,